.github/workflows/release.yml

```diff
@@ -0,0 +1,111 @@
+name: Release
+
+on:
+  push:
+    tags:
+    - 'v*'
+
+jobs:
+
+  download-uberjar:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 10
+    steps:
+    - name: Download Uberjar for ${{ github.ref_name }}
+      run: |
+        JAR_DOWNLOAD_URL=https://downloads.metabase.com/${{ github.ref_name }}/metabase.jar
+        echo $JAR_DOWNLOAD_URL > url.txt
+        echo "----- Downloading Uberjar from $JAR_DOWNLOAD_URL -----"
+        curl -OL $JAR_DOWNLOAD_URL
+        stat ./metabase.jar
+        date | tee timestamp
+    - name: Calculate SHA256 checksum
+      run: sha256sum ./metabase.jar | tee SHA256.sum
+    - name: Upload Uberjar as artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: metabase-uberjar-${{ github.ref_name }}
+        path: |
+          ./metabase.jar
+          ./url.txt
+          ./timestamp
+          ./SHA256.sum
+
+  check-uberjar:
+    runs-on: ubuntu-20.04
+    needs: download-uberjar
+    timeout-minutes: 10
+    steps:
+    - name: Prepare JRE (Java Run-time Environment)
+      uses: actions/setup-java@v1
+      with:
+        java-package: jre
+        java-version: 11
+    - uses: actions/download-artifact@v2
+      name: Retrieve previously downloaded Uberjar
+      with:
+        name: metabase-uberjar-${{ github.ref_name }}
+    - name: Reveal its version.properties
+      run: jar xf metabase.jar version.properties && cat version.properties
+    - name: Display when and where it was downloaded
+      run: |
+        cat timestamp
+        cat url.txt
+    - name: Show the checksum
+      run: cat SHA256.sum
+    - name: Launch Metabase Uberjar (and keep it running)
+      run: java -jar ./metabase.jar &
+    - name: Wait for Metabase to start
+      run: while ! curl -s localhost:3000/api/health; do sleep 1; done
+      timeout-minutes: 3
+    - name: Check API health
+      run: curl -s localhost:3000/api/health
+
+  containerize:
+    runs-on: ubuntu-20.04
+    needs: check-uberjar
+    timeout-minutes: 15
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/download-artifact@v2
+      name: Retrieve previously downloaded Uberjar
+      with:
+        name: metabase-uberjar-${{ github.ref_name }}
+    - name: Move the Uberjar to the context dir
+      run: mv ./metabase.jar bin/docker/.
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
```

Comment:
The latest version is 2, should we use it instead ?

Reply:
`uberjar.yml` uses v1 so I think we should stick with it, since we know it's working.

Later on if necessary, we can bump all `setup-buildx-action` to other version to check how it performs.